addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3188,7 +3188,24 @@ export class OdooEditor extends EventTarget {
         this.deleteRange();
     }
     _onClipboardCopy(clipboardEvent) {
-        if (this.isSelectionInEditable()) {
+        const hasPartiallySelectedTable = hasTableSelection(this.editable) &&
+            ![...this.editable.querySelectorAll('.o_selected_table td')].every(td => (
+                !closestElement(td, 'table').classList.contains('o_selected_table') || // ignore nested
+                td.classList.contains('o_selected_td')
+            ));
+        if (hasPartiallySelectedTable) {
+            clipboardEvent.preventDefault();
+            const table = this.editable.querySelector('.o_selected_table');
+            const copyTable = table.cloneNode();
```

Comment:
Shouldn't it be `tableCopy` then ? My first thought when I read the variable was that it was going to be a function that copies a table then I was surprised when I looked at the right-hand side of the assignment. While we're at it, I suggest `tableClone` instead.